website_sale_product_contract_gift/views/templates.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template id="payment" inherit_id="website_sale.payment">
+        <xpath expr="//div[@id='payment_method']" position="before">
+            <div t-if="website_sale_order.is_gift" id="order_is_gift" class="mt-3">
+                <h3>This order is a gift</h3>
+                <p>
+                    <b>Date of the gift:</b> <span
+                        t-field="website_sale_order.gift_date"
+                    />
```

Comment:
what is the difference between `gift_date` and `date_for_gift`?

Reply:
None